rootfs/rootfs/usr/local/bin/autologin

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
@tianon did you add this file on purpose?
I already have one with boot2docker 1.12.5 which contains the following
```
$ cat /usr/local/bin/autologin
#!/bin/sh
/bin/login -f docker
```